sale_order_line_variant_description/tests/test_sale_order_line_variant_description.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2015-17 Alex Comba - Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests import Form, TransactionCase
+
+
+class TestSaleOrderLineVariantDescription(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.fiscal_position_model = cls.env["account.fiscal.position"]
+        cls.tax_model = cls.env["account.tax"]
+        cls.pricelist_model = cls.env["product.pricelist"]
+        cls.uom_uom_model = cls.env["uom.uom"]
+        cls.product_tmpl_model = cls.env["product.template"]
+        cls.product_model = cls.env["product.product"]
+        cls.so_model = cls.env["sale.order"]
+        cls.so_line_model = cls.env["sale.order.line"]
+        cls.partner = cls.env.ref("base.res_partner_1")
+
+    def test_product_id_change(self):
+        uom = self.uom_uom_model.search([("name", "=", "Units")])[0]
+
+        tax_form = Form(self.tax_model)
```

Comment:
Even though I usually recommend the `Form` class whenever possible, this is accounting data, it's always created without it. We're not testing the accounting flow in this module. When creating orders and lines is more reasonable because it's part of the tested flow.

BTW, do you think this could be removed? AFAIK creating taxes and fiscal positions is not necessary for thse test cases to work.